xblockmufi/private/view.js

```diff
@@ -4,8 +4,91 @@ function XblockMufiView(runtime, element) {
     var $ = window.jQuery;
     var $element = $(element);
 
-    // TODO: Put your logic here
+    var handlerUrl = runtime.handlerUrl(element, 'student_submit');
+	var publishUrl = runtime.handlerUrl(element, 'publish_event');
 
-    // To find elements inside your XBlock, try:
-    // var myElement = $element.find('.myElement');
+    var submit_button = $element.find('.submit_button');
+    var reset_button = $element.find('.reset_button');
+
+    var your_answer = $element.find('.your_answer');
+    var expert_answer = $element.find('.expert_answer');
+
+    function publish_event(data) {
+      $.ajax({
+          type: "POST",
+          url: publishUrl,
+          data: JSON.stringify(data)
+      });
+    }
+
+	function post_submit(result) {
```

Comment:
it's empty right now, but I think I'll need to use it to fix the multi unit issue that I think is present in this version of the XBlock.